src/main/java/org/fundacion/jala/converter/controller/config/SwaggerConfig.java

```diff
@@ -26,24 +29,58 @@
 public class SwaggerConfig {
 
     /**
-     * Creates a Docket.
+     * Creates the API Docket.
      *
      * @return a Docket with Swagger configuration.
      */
     @Bean
     public Docket api() {
         return new Docket(DocumentationType.SWAGGER_2)
+                .groupName("API")
```

Comment:
It should be in a line until 120 characteres.

Reply:
solved it.